src/frontend/src/btc/services/worker.btc-wallet.services.ts

```diff
@@ -48,8 +47,12 @@ export const initBtcWalletWorker = async ({
 				syncWalletError({
 					tokenId,
 					error: (data.data as PostMessageDataResponseError).error,
-					// TODO: do not launch worker locally if BTC canister is not deployed, and remove the below param afterwards
-					hideToast: isRegtestNetwork && LOCAL
+					/**
+					 * TODOs:
+					 * 1. Do not launch worker locally if BTC canister is not deployed, and remove "isRegtestNetwork" afterwards.
+					 * 2. Wait for testnet BTC canister to be fixed on the IC side, and remove "isTestnetNetwork" afterwards.
+					 * **/
+					hideToast: isRegtestNetwork || isTestnetNetwork
```

Comment:
I removed `LOCAL` since it is not really needed here - `isRegtestNetwork` can only be launched locally.

Reply:
Just pinging @peterpeterparker to be in the loop - we discussed this part previously, and agreed not to hide errors outside of localhost, but it seems we are forced to have an additional check to avoid having toasts shown every time someone loads BTC testnet wallet.